packages/shared/lib/services/sync/sync.service.ts

```diff
@@ -184,37 +182,29 @@ export const getSyncs = async (
                             'sync_config_id', ${SYNC_JOB_TABLE}.sync_config_id,
                             'version', ${SYNC_CONFIG_TABLE}.version,
                             'models', ${SYNC_CONFIG_TABLE}.models
-                            )
-                        END as latest_sync `),
-                    db.knex.raw(
-                        `ROW_NUMBER() OVER (PARTITION BY ${TABLE}.id, ${SYNC_JOB_TABLE}.sync_id ORDER BY ${SYNC_JOB_TABLE}.updated_at DESC) as job_row_number`
-                    )
-                )
-                .leftJoin(ACTIVE_LOG_TABLE, function () {
-                    this.on(`${ACTIVE_LOG_TABLE}.sync_id`, `${TABLE}.id`)
-                        .andOnVal(`${ACTIVE_LOG_TABLE}.active`, true)
-                        .andOnVal(`${ACTIVE_LOG_TABLE}.type`, 'sync');
-                })
-                .join(SYNC_CONFIG_TABLE, function () {
-                    this.on(`${SYNC_CONFIG_TABLE}.sync_name`, `${TABLE}.name`)
-                        .andOn(`${SYNC_CONFIG_TABLE}.deleted`, '=', db.knex.raw('FALSE'))
-                        .andOn(`${SYNC_CONFIG_TABLE}.active`, '=', db.knex.raw('TRUE'))
-                        .andOn(`${SYNC_CONFIG_TABLE}.type`, '=', db.knex.raw('?', 'sync'));
-                })
-                .leftJoin(SYNC_JOB_TABLE, function () {
-                    this.on(`${SYNC_JOB_TABLE}.sync_id`, `${TABLE}.id`)
-                        .andOn(db.knex.raw(`${SYNC_JOB_TABLE}.deleted = FALSE`))
-                        .andOn(db.knex.raw(`${SYNC_JOB_TABLE}.sync_config_id IS NOT NULL`));
-                })
-                .where({
-                    nango_connection_id: nangoConnection.id,
-                    [`${SYNC_CONFIG_TABLE}.nango_config_id`]: nangoConnection.config_id,
-                    [`${TABLE}.deleted`]: false
-                })
-                .orderBy(`${TABLE}.name`, 'asc');
+                        )
+                        FROM ${SYNC_JOB_TABLE}
+                        JOIN ${SYNC_CONFIG_TABLE} ON ${SYNC_CONFIG_TABLE}.id = ${SYNC_JOB_TABLE}.sync_config_id AND ${SYNC_CONFIG_TABLE}.deleted = false
+                        WHERE ${SYNC_JOB_TABLE}.sync_id = ${TABLE}.id AND ${SYNC_JOB_TABLE}.deleted = false
+                        ORDER BY ${SYNC_JOB_TABLE}.created_at DESC
```

Comment:
this order by is the actual fix so this subquery can avoid a expensive `sort`.
Using `created_at` or `updated_at` is equivalent in term of getting the most recent job